scripts/clean-local-binary.sh

```diff
@@ -2,7 +2,7 @@
 
 # no `set -e` here as we allow commands to fail in this script
 
-TMPDIR=/tmp/parachain_dev
+TMPDIR=${TMPDIR:-"/tmp/parachain_dev"}
```

Comment:
`TMPDIR=${TMPDIR:-"/tmp/parachain_dev"}` where does TMPDIR value come from when it's not default case?

Reply:
It is set from `launch.py` when pass it in as a CLI parameter